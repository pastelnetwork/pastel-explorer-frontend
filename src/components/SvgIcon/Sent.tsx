const Sent = () => {
  return (
    <svg viewBox="0 0 69 65" fill="none" xmlns="http://www.w3.org/2000/svg">
      <path
        d="M31.8959 45.75C31.8959 50.3913 33.7396 54.8425 37.0215 58.1244C40.3034 61.4063 44.7546 63.25 49.3959 63.25C54.0372 63.25 58.4884 61.4063 61.7702 58.1244C65.0521 54.8425 66.8959 50.3913 66.8959 45.75C66.8959 41.1087 65.0521 36.6575 61.7702 33.3756C58.4884 30.0937 54.0372 28.25 49.3959 28.25C44.7546 28.25 40.3034 30.0937 37.0215 33.3756C33.7396 36.6575 31.8959 41.1087 31.8959 45.75Z"
        stroke="#E94830"
        strokeWidth="3"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M2 10.75C2 13.0706 3.84374 15.2962 7.12563 16.9372C10.4075 18.5781 14.8587 19.5 19.5 19.5C24.1413 19.5 28.5925 18.5781 31.8744 16.9372C35.1563 15.2962 37 13.0706 37 10.75C37 8.42936 35.1563 6.20376 31.8744 4.56282C28.5925 2.92187 24.1413 2 19.5 2C14.8587 2 10.4075 2.92187 7.12563 4.56282C3.84374 6.20376 2 8.42936 2 10.75V10.75Z"
        stroke="#E94830"
        strokeWidth="3"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M37 19.5C37 24.3329 29.1658 28.25 19.5 28.25C9.83417 28.25 2 24.3329 2 19.5"
        stroke="#E94830"
        strokeWidth="3"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M37 23.875V10.75"
        stroke="#E94830"
        strokeWidth="3"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M2 10.75V54.5C2 59.3329 9.83417 63.25 19.5 63.25C21.7038 63.2535 23.9023 63.0356 26.0625 62.5996"
        stroke="#E94830"
        strokeWidth="3"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M2 45.75C2 50.5829 9.83417 54.5 19.5 54.5C21.7038 54.5035 23.9023 54.2856 26.0625 53.8496"
        stroke="#E94830"
        strokeWidth="3"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M2 37C2 41.8329 9.83417 45.75 19.5 45.75C21.7038 45.7535 23.9023 45.5356 26.0625 45.0996"
        stroke="#E94830"
        strokeWidth="3"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M2 28.25C2 33.0829 9.83417 37 19.5 37C21.7038 37.0035 23.9023 36.7856 26.0625 36.3496"
        stroke="#E94830"
        strokeWidth="3"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M50.4494 38.9161C50.2852 38.9184 50.1286 38.9853 50.0133 39.1023C49.898 39.2192 49.8334 39.3769 49.8334 39.5411V42.7849C48.726 42.9084 46.9054 43.2258 45.0726 44.4125C42.9917 45.7599 41.1418 48.237 41.0988 52.3267C41.0982 52.3298 41.0962 52.3317 41.0956 52.3349H41.0972C41.0968 52.3798 41.0834 52.4124 41.0834 52.4577C41.0835 52.6127 41.1413 52.7621 41.2454 52.8769C41.3494 52.9917 41.4925 53.0638 41.6467 53.0791C41.8009 53.0944 41.9553 53.0519 42.0799 52.9598C42.2046 52.8677 42.2905 52.7325 42.3212 52.5806C42.8159 50.1072 44.358 49.0111 46.0671 48.449C47.5096 47.9745 48.9219 47.9744 49.8334 48.0225V51.2077C49.8334 51.3313 49.8701 51.4521 49.9388 51.5549C50.0075 51.6576 50.1051 51.7377 50.2192 51.785C50.3334 51.8323 50.4591 51.8447 50.5803 51.8206C50.7015 51.7965 50.8128 51.737 50.9003 51.6496L56.7336 45.8163C56.8508 45.6991 56.9166 45.5401 56.9166 45.3744C56.9166 45.2087 56.8508 45.0497 56.7336 44.9325L50.9003 39.0992C50.8412 39.0401 50.7708 38.9934 50.6934 38.962C50.616 38.9305 50.533 38.9149 50.4494 38.9161V38.9161ZM51.0834 41.0499L55.4079 45.3744L51.0834 49.699V47.4097C51.0834 47.2541 51.0254 47.104 50.9207 46.9888C50.8159 46.8737 50.6719 46.8018 50.517 46.7872C49.6487 46.7059 47.6624 46.6084 45.6765 47.2616C44.7487 47.5668 43.8273 48.0804 43.0365 48.8144C43.6694 47.2668 44.6441 46.1798 45.7522 45.4623C47.5746 44.2823 49.6689 43.9954 50.5039 43.934C50.6614 43.9225 50.8087 43.8518 50.9161 43.7361C51.0236 43.6205 51.0834 43.4685 51.0834 43.3106V41.0499Z"
        fill="#E94830"
      />
    </svg>
  );
};

export default Sent;
