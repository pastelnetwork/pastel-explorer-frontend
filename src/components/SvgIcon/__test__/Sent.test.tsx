import { shallow } from 'enzyme';

import Sent from '../Sent';

describe('components/SvgIcon/Sent', () => {
  const wrapper = shallow(<Sent />);

  test('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should render <svg>', () => {
    expect(wrapper.find('svg').length).toBe(1);
  });

  test('should render <path>', () => {
    expect(wrapper.find('path').length).toBe(9);
  });
});
