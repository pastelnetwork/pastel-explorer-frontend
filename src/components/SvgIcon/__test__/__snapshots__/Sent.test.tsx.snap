// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/SvgIcon/Sent renders correctly 1`] = `ShallowWrapper {}`;
